DOCUMENTATION.md

```diff
@@ -216,10 +216,17 @@ You can leave out this argument if your service only has one reply-to email addr
 
 To send a file by email, add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file.
 
-The file will be available for the recipient to download for 18 months.
-
 The links are unique and unguessable. GOV.UK Notify cannot access or decrypt your file.
 
+Your file will be available to download for a default period of 78 weeks (18 months). From 29 March 2023 we will reduce this to 26 weeks (6 months) for all new files. Files sent before 29 March are not affected.
+
+To help protect your files you can also:
+
+* ask recipients to confirm their email address before downloading
```

Comment:
Optional: will this sentence need changing in 6 months time? Maybe in 6 months time I'd phrase it more like we ask recipients to confirm their email address before downloading. You can choose to turn this off. 

I think for speed of getting this merged, I think we don't make a change on this for the moment at least.

Reply:
Yes, the whole section will need a little makeover in 6 months.

All the ‘turn it on’ stuff will go away, the retention period will be fixed at 26 weeks, etc.